### PR TITLE
Updated dependencies and update to 1.17.1

### PR DESCRIPTION
**Pull request by Chris6ix**
Will also update this dependency in FlyPerms when this is merged ;)

### CODE REVIEW

**Comment by benwoo1110** [comment]
Oh forgot ot mention, groupId for paper is changed to `io.papermc.paper`. Had to change that just now.

Edit: also is paper for 1.17.1 out yet tho?

**Comment by Chris6ix** [comment]
No Paper 1.17.1 isn't out yet, but will probably not take long.
Or did you want to release an 1.17 Update now and then 1.17.1 later?
My thought was to prepare the API and plugin for 1.17.1 and then release it when Paper 1.17.1 is out...

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>dev.benergy10.minecrafttools</groupId>
     <artifactId>MinecraftTools</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.1-SNAPSHOT</version>
```
Comment (benwoo1110) [inline pom.xml]: can keep this, doesn't have api breakage

**Comment by benwoo1110** [comment]
I dont think anything else needs to change tbh, just good to update dependency for sure.

File: pom.xml (hunk 1 above)
Comment (Chris6ix) [reply pom.xml]: okey

**Comment by Chris6ix** [comment]
Alright

File: pom.xml [hunk 2]
```diff
@@ -145,7 +145,7 @@
         <dependency>
             <groupId>com.destroystokyo.paper</groupId>
```
Comment (benwoo1110) [inline pom.xml]:
```suggestion
            <groupId>io.papermc.paper</groupId>
```

**Review by benwoo1110 — APPROVED** [review]
1.17.1 is out, so should be safe to merge. Thanks!

**Pull request merged**